app/screens/home/search/search.tsx

```diff
@@ -304,15 +310,15 @@ const SearchScreen = ({teamId, teams}: Props) => {
     const onFlatLayout = useCallback(() => {
         if (clearRef.current || cancelRef.current) {
             unlock();
+            onSnapWithTimeout(0);
```

Comment:
Maybe this is all I needed the entire time?

Reply:
I tried with just this, but on iOS, it still remains an issue with frame freezing during animation while keyboard appear/dismis.  This is causing the header view to sometimes ended up having different height, depending what value of e.contentOffset.y is being assigned into scrollValue.

So the changes to lockValue (using state vs animated value) is still pertinent as it makes the header to have a much proper size.